src/w3d/renderer/render2dsentence.cpp

```diff
@@ -845,7 +853,7 @@ void Render2DSentenceClass::Build_Sentence_Centered(const unichar_t *text, int *
 
             if (m_processAmpersand && cur_ch == U_CHAR('&')) {
                 unichar_t next_ch = *text;
-                if (next_ch != U_CHAR('\0') && next_ch > U_CHAR(' ')) {
+                if (next_ch != U_CHAR('\0') && next_ch > U_CHAR(' ') && next_ch != U_CHAR('\n')) {
```

Comment:
Looks like `next_ch != U_CHAR('\n')` can be omitted because of `next_ch > U_CHAR(' ')`. It's odd this was compiled in though. Hmm.

Reply:
Fixed.